### PR TITLE
fix #6220 --no-builds flag for 'conda env export'

### LINKED ISSUE #6220: Add flag to build environment.yml without build strings
https://gitter.im/conda/conda?at=59ef54ebe44c43700a70e9a4
https://twitter.com/drvinceknight/status/922837449092542464?ref_src=twsrc%5Etfw

> Due to hashes of packages being introduced in `envinronment.yml` I'm getting all sorts of issues with building envs from file. (Very new problem)

### PR DESCRIPTION
**Pull request by kalefranz**
fix #6220

### CODE REVIEW

**Comment by kalefranz** [comment]
flake8 is expected to fail here until we merge https://github.com/conda/conda/pull/6205

**Comment by kalefranz** [comment]
cherry-picked 53c20db from https://github.com/conda/conda/pull/6154

File: tests/conda_env/test_cli.py [hunk 1]
```diff
@@ -243,7 +244,7 @@ def test_create_env(self):
         run_conda_command(Commands.CREATE, test_env_name_2)
         self.assertTrue(env_is_created(test_env_name_2))
 
-    def test_export(self):
+    def test_export_1(self):
```
Comment (nehaljwani) [inline tests/conda_env/test_cli.py]: What does the _1 denote here?

Comment (kalefranz) [reply tests/conda_env/test_cli.py]: To distinguish from https://github.com/conda/conda/pull/6221/files#diff-771b53ed2768cb3fe7eb0bc279a57c34R297

That is, with

    py.test tests/conda_env/test_cli.py -k test_export

there was no way to specify only the one test before.

Comment (kalefranz) [reply tests/conda_env/test_cli.py]: Did that make any sense at all? 🙃

Comment (kalefranz) [reply tests/conda_env/test_cli.py]: Now, with 

    py.test tests/conda_env/test_cli.py -k test_export

you get the two tests, but if you want to target that single test, 

        py.test tests/conda_env/test_cli.py -k test_export_1

So it's just about name uniqueness.

Comment (nehaljwani) [reply tests/conda_env/test_cli.py]: Ah, okay.

Comment (nehaljwani) [reply tests/conda_env/test_cli.py]: Well, I wouldn't be too pedantic and ask it to be renamed to `test_env_export` or something.

Comment (kalefranz) [reply tests/conda_env/test_cli.py]: There ya go :)

**Pull request merged**